### PR TITLE
* [doc] add Longpress event to common-event.md

### PR DESCRIPTION
**Pull request by vczero**
1.  add Longpress event introduction.
2.  add example code.

### CODE REVIEW

File: doc/references/common-event.md [hunk 1]
```diff
@@ -12,7 +12,50 @@ If a `click` event is bound to a component, the event will be triggered when the
 
 * `type`: `click`
 * `target`: The target component where the event is triggered
-* `timestamp`: Timestamp when event is triggered.
+* `timestamp`: Timestamp when event is triggered.       
+
+## Longpress event      
+
+If a `longpree` event is bound to a component, the event will be triggered when the user long press on it.
```
Comment (luics) [inline doc/references/common-event.md]: `longpree` -> `longpress`

**Comment by luics** [comment]
@YorkShen `longpress` can be accessed in ios/android/html5 ?

**Comment by YorkShen** [comment]
It works fine on Android and iOS.

**Comment by vczero** [comment]
@luics  I fix the  word. longpree -> longpress

**Comment by luics** [comment]
@MrRaindrop is html5 ready?

**Comment by MrRaindrop** [comment]
it works fine @luics

**Pull request merged**